src/Microsoft.AspNetCore.Mvc.Core/Internal/ObjectResultExecutor.cs

```diff
@@ -65,6 +66,11 @@ public class ObjectResultExecutor
         protected bool RespectBrowserAcceptHeader { get; }
 
         /// <summary>
+        /// Gets the value of <see cref="MvcOptions.ReturnHttpNotAcceptable"/>.
+        /// </summary>
+        protected bool ReturnHttpNotAcceptable { get; }
```

Comment:
I'm not sure that they need to be `protected`; I just made the new property `protected` to match what was there :smile_cat: